packages/vue/src/components/atoms/SfImage/SfImage.vue

```diff
@@ -80,6 +88,43 @@ export default {
       default: "lazy",
       validator: (value) => ["", "lazy", "eager"].includes(value),
     },
+    imageTag: {
+      type: String,
+      default: "img",
+      validator: (value) => ["img", "nuxt-img", "nuxt-picture"].includes(value),
+    },
+    sizes: {
+      type: String,
+      default: "",
+    },
+    provider: {
+      type: String,
+      default: "",
+    },
+    preset: {
+      type: String,
+      default: "",
+    },
+    format: {
+      type: String,
+      default: "",
+    },
+    quality: {
+      type: Number,
+      default: null,
+    },
+    fit: {
+      type: String,
+      default: "",
```

Comment:
what do you think about changing props default values to `null`?

Reply:
suggestion: maybe we should also add required conditional if image is "nuxt-img"?

Reply:
I added `missing prop fit` error showing in the console when there is no prop fit added.